### PR TITLE
[API] Show SRI data

### LINKED ISSUE #190: SRI data in API
Current API doesn't provide SRI related data, should add.

### PR DESCRIPTION
**Pull request by agathver**
Closes #190

Adds a key `sri` to assets, if the parameter `includeSri=true` is added to a library listing

### CODE REVIEW

File: apiServer.js [hunk 1]
```diff
@@ -140,6 +141,16 @@ app.get('/libraries/:library', function(req, res) {
     });
     results[0] = ret;
   }
+  if (includeSri && results[0].assets) {
+    _.each(results[0].assets, function(asset) {
+      console.log('sri/' + req.params.library + '/' + asset.version + '.json');
```
Comment (PeterDaveHello) [inline apiServer.js]: Can we remove this line or add more useful hint with it?

**Comment by PeterDaveHello** [comment]
@agathver almost done 👍 Do you mind to update the doc about this part in file `templates/api.html`? So I can merge it 😄

**Comment by PeterDaveHello** [comment]
@agathver any chance you'd still like to update this PR? Thank you.

**Comment by PeterDaveHello** [comment]
Let me know if you'd need help, thanks!

**Comment by MattIPv4** [comment]
Closing as duplicate (#246).

**Pull request closed without merging**